packages/hierarchies-react/src/presentation-hierarchies-react/internal/TreeLoader.ts

```diff
@@ -27,6 +27,7 @@ export interface LoadNodesOptions {
   shouldLoadChildren: (node: TreeModelHierarchyNode) => boolean;
   buildNode?: (node: TreeModelHierarchyNode) => TreeModelHierarchyNode;
   ignoreCache?: boolean;
+  discardState?: boolean;
```

Comment:
It doesn't look like it belongs here. Why not place the attribute on `LoadDebouncedNodesOptions` instead?

Reply:
moved.